lib/new_relic/telemetry/finch.ex

```diff
@@ -130,9 +130,9 @@ defmodule NewRelic.Telemetry.Finch do
           category: "http",
           attributes:
             %{
-              "request.url": url,
-              "request.method": request.method,
-              "request.client": "Finch",
+              url: url,
```

Comment:
Just out of curiosity, where did you find that the correct attributes are `url` and no longer `request.url`? I feel like I've tried looking for those docs before and failed 🙃

Reply:
The `set_span` function has function heads for the span types that the UI knows about. `url` turns into `http.url` actually

Reply:
This is where the knowledge of the per-"category" attributes that the UI knows about lives:

https://github.com/newrelic/elixir_agent/blob/master/lib/new_relic/span/event.ex#L70-L73

I don't know if it's documented anywhere. Also, it's always been "http.url", I just forgot that there's special attributes that are needed when I wrote the Finch instrumentation.

Reply:
I found it: 
- https://docs.newrelic.com/docs/distributed-tracing/trace-api/trace-api-decorate-spans-attributes/#decorate